http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/EnabledUserDirectives.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.dsde.workbench.leonardo.http
+package api
+
+import akka.http.scaladsl.server.Directive
+import org.broadinstitute.dsde.workbench.model.UserInfo
+
+trait EnabledUserDirectives {
```

Comment:
This and some of the other plumbing for these new Directives are cargo-cult copies of UserInfoDirectives. I don't immediately see why this or UserInfoDirectives should be a `trait` since it's passed as a param into HttpRoutes by Boot.scala. Orch uses `with *Directives`; maybe that's better use of the architecture.